README.md

```diff
@@ -22,10 +22,10 @@ $ symfony-upgrade-fixer fix /path/to/code --dry-run
 | Name  | Description |
 | ----  | ----------- |
 | property_access | Renamed PropertyAccess::getPropertyAccessor to PropertyAccess::createPropertyAccessor. |
+| inherit_data_aware_iterator | The class VirtualFormAwareIterator was renamed to InheritDataAwareIterator. |
```

Comment:
Actually, I think it comes from the finder. Not the alpha sorted ;) so it will be stable.